Analysis of asthma mAbs

AnnaB45 opened the pull request:
Analysis of the use of asthma mAbs in secondary care, using R.
R file for exploritary analysis, R markdown file for writing up analysis and reference data for mapping trusts to stps and regions.

AnnaB45 committed:
Analysis of asthma mAbs

Analysis of the use of asthma mAbs in secondary care, using R.
R file for exploritary analysis, R markdown file for writing up analysis and reference data for mapping trusts to stps and regions.

wjchulme committed:
add Rproject and update directories with here::here

wjchulme committed:
simplify library calls

remove dplyr, scales, ggplot2 as these are all shipped with tidyverse

wjchulme committed:
refactor ifelse with case_when

wjchulme committed:
minor aesthetic changes

wjchulme committed:
removed library(reshape)

could also change `dplyr::rename` to `rename` since there's no longer a conflict with reshape

wjchulme reviewed (approved):
Looks great! I've made some minor changes to make the code/graphs a bit clearer but not spotted anything untoward. 

It probably worth 
1. linking to wherever NHSD host the actual data 
2. explaining that this data has been processed and put in a (restricted access?) database by the datalab
3. explaining that to re-run the code you'll need to figure out credentials (eg use https://gargle.r-lib.org/articles/non-interactive-auth.html for knitting the markdown)

I don't have a comprehensive knowledge of trust mergers but I do happen to know that University Hospital of South Manchester NHS FT no longer exists. Does this need to be updated?

AnnaB45 committed:
Reknit markkdown and changed figure size

Installed "here" library on machine, reran R markdown code and changed width of all figures so uniform

wjchulme reviewed (commented):
`scales`, `dplyr` and `ggplot2` all ship with `tidyverse` so these don't need to be explicitly installed. `dplyr` and `ggplot2` are loaded with `library(tidyverse)` so don't need to be explicitly loaded.

But it's just neater, doesn't make any difference to code execution!
